src/taco-remote-lib/common/builder.ts

```diff
@@ -132,28 +132,19 @@ class Builder {
             fs.mkdirSync("platforms");
```

Comment:
Might be worth changing this function name to `addAndPreparePlatform` to be more accurate.